src/components/ServiceHealth/HealthDetails.tsx

```diff
@@ -36,28 +38,39 @@ export class HealthDetails extends React.PureComponent<Props, {}> {
     const globalDeplStatus: H.Status = allInactive
       ? H.FAILURE
       : deplsStatuses.reduce((prev, cur) => H.mergeStatus(prev, cur), H.NA);
+    const reqErrorsRatio = H.getRequestErrorsRatio(health.requests);
+    const reqErrorsStatus = H.requestErrorsThresholdCheck(reqErrorsRatio);
     return (
       <Tooltip id={this.props.id + '-health-tooltip'}>
         <div style={{ paddingLeft: 15, paddingRight: 15, paddingBottom: 10 }}>
           <div>
             <h4>{this.props.headline}</h4>
           </div>
           <div>
-            <strong>{this.renderStatus(globalDeplStatus)}&nbsp; Deployments status:</strong>
+            <strong>
+              {this.renderStatus(globalDeplStatus)}
+              {' Deployments Status:'}
+            </strong>
             <ul style={{ listStyleType: 'none', paddingLeft: 12 }}>
               {health.deploymentStatuses.map((st, idx) => {
                 return (
                   <li key={idx}>
-                    {this.renderStatus(deplsStatuses[idx])}&nbsp;
-                    {st.name} ({st.available} / {st.replicas})
+                    {this.renderStatus(deplsStatuses[idx])} {st.name}: {st.available} / {st.replicas}
                   </li>
                 );
               })}
             </ul>
             <strong>
-              {this.renderStatus(H.ratioCheck(health.envoy.healthy, health.envoy.total))}&nbsp; Envoy health
+              {this.renderStatus(H.ratioCheck(health.envoy.healthy, health.envoy.total))}
+              {' Envoy Health:'}
+            </strong>
+            {' ' + health.envoy.healthy + ' / ' + health.envoy.total}
```

Comment:
Just a refresh to confirm, envoy healthy gets a "healthy / non healthy" status from the envoy proxy and this is a ratio of all proxies status for a given service.

Reply:
yes, or in terms of pods , normally envoy.total == total number of running deployment pods in a service